roles/create-openshift-resources/tasks/promote_image.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Sure. This gets into the question of the service user. Now that we have IDM, questions of a identity and RBAC matter, where as previously they did not.